code/modules/supply/supply_packs/pack_science.dm

```diff
@@ -5,6 +5,34 @@
 	containertype = /obj/structure/closet/crate/sci
 	department_restrictions = list(DEPARTMENT_SCIENCE)
 
+/datum/supply_packs/science/rnd	// Everything you need to kick-start Science from scratch once the dust of the apocalypse has blown over.
+	name = "Research & Development Crate"
+	contains = list(/obj/item/circuitboard/rdconsole,
+					/obj/item/circuitboard/protolathe,
+					/obj/item/circuitboard/destructive_analyzer,
+					/obj/item/circuitboard/circuit_imprinter,
+					/obj/item/stock_parts/manipulator,
+					/obj/item/stock_parts/manipulator,
+					/obj/item/stock_parts/manipulator,
+					/obj/item/stock_parts/manipulator,
+					/obj/item/stock_parts/matter_bin,
+					/obj/item/stock_parts/matter_bin,
+					/obj/item/stock_parts/matter_bin,
+					/obj/item/stock_parts/scanning_module,
+					/obj/item/stock_parts/micro_laser,
+					/obj/item/reagent_containers/glass/beaker,
+					/obj/item/reagent_containers/glass/beaker,
+					/obj/item/reagent_containers/glass/beaker,
+					/obj/item/reagent_containers/glass/beaker,
```

Comment:
maybe these could get shoved in a wrapped box or something so that 12 stock parts don't bloat the crate when first opened.

Reply:
I have shoved the entire kit into a large box.